webapp/channels/src/i18n/en.json

```diff
@@ -109,7 +110,7 @@
   "add_command.url.placeholder": "Must start with http:// or https://",
   "add_command.urlRequired": "A request URL is required",
   "add_command.username": "Response Username",
-  "add_command.username.help": "(Optional) Specify the name to use when posting responses for this slash command. Usernames can be up to 22 characters, and contain lowercase letters, numbers, and the symbols \"-\", \"_\", and \".\". If left blank, your Mattermost username is used.",
+  "add_command.username.help": "(Optional) Specify the name to use when posting responses for this slash command. Usernames can be up to 22 characters, and contain lowercase letters, numbers, and the symbols \\\"-\\\", \\\"_\\\", and \\\".\\\". If left blank, your Mattermost username is used.",
```

Comment:
There's too many backslashes here, probably because of that escaping thing you mentioned a couple weeks back